File: backend/src/main/java/pl/pamsoft/ebs/dto/SimulatedEstimation.java
package pl.pamsoft.ebs.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class SimulatedEstimation {
	Integer min;
	Integer max;
	Integer avg;
	Integer count;
	Integer originalEstimation;
	List<Integer> values;
}

File: frontend/src/app/model/SimulatedEstimation.ts

import {Person} from "./Person";
import {Task} from "./Task";
export class SimulatedEstimation {
  constructor(public min: number,
              public max: number,
              public avg: number,
              public count: number,
              public originalEstimation: number,
              public values: number[]) {
  }

}
